cmd/mattermost/commands/integrity.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/store"
+	"github.com/spf13/cobra"
+)
+
+var IntegrityCmd = &cobra.Command{
+	Use:   "integrity",
+	Short: "Check database data integrity",
+	RunE:  integrityCmdF,
+}
+
+func init() {
+	IntegrityCmd.Flags().Bool("confirm", false, "Confirm you really want to run a complete integrity check of your DB.")
```

Comment:
Probably here would be nice to know why are you asking me to confirm (like in the confirmation text that you added bellow)

Reply:
That's a good idea. Especially since that line is documenting the command.